kahuna/public/js/image/controller.js

```diff
@@ -41,6 +48,9 @@ image.controller('ImageCtrl', [
         };
 
         ctrl.image = image;
+
+        ctrl.imageDimensions = imageService(ctrl.image).dimensions;
```

Comment:
Not too important here, but I may be more keen on having dedicated accessor helpers as we have in Composer (e.g. `imageAccessor.getDimensions(image)`) rather than one function like here. The function here does a lot of work computing lots of information that isn't required, making this slower than necessary, and also harder to track what code uses what computed information.